_data/navigation.js

```diff
@@ -261,14 +260,18 @@ module.exports = {
             title: 'Manage your Upkeeps',
             url: '/docs/chainlink-keepers/manage-upkeeps/',
           },
+          {
+            title: 'Making Flexible Contracts',
```

Comment:
Where should this tutorial live in the docs? I'm leaning towards the Getting Started portion. Thoughts?

Reply:
I think it would be more consistent for the users to keep it under keepers docs. The structure would be the following:

- Intro about keepers
- Basic tutorial (counter example)
- More advanced one (how to use checkdata and performdata example)
- Utility contracts (with examples)
- token economics...Etc

Anyone who needs to learn about keepers will get his info from: https://docs.chain.link/docs/chainlink-keepers/*

Reply:
This could go in the beginners guide but this guide points to the other Keepers guides as prerequisites.

Let's keep this as it is for now and add a more simple keepers example in the beginner's guide.